sites/org/pages/newsletter/[dir].mjs

```diff
@@ -0,0 +1,62 @@
+import { nsMerge, localePath } from 'shared/utils.mjs'
+import { pages as posts } from 'site/prebuild/newsletter.posts.mjs'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { loadMdxAsStaticProps } from 'shared/mdx/load.mjs'
+import { PageWrapper, ns as pageNs } from 'shared/components/wrappers/page.mjs'
+import { PostLayout, ns as layoutNs } from 'site/components/layouts/post.mjs'
+
+const ns = nsMerge(pageNs, layoutNs)
+
+const NewsletterPage = ({ dir, page, mdx, frontmatter }) => {
+  return (
+    <PageWrapper
+      {...page}
+      title={frontmatter.title}
+      intro={frontmatter.intro}
+      layout={(props) => (
+        <PostLayout
+          {...props}
+          slug={page.path.join('/')}
+          frontmatter={frontmatter}
+          mdx={mdx}
+          dir={dir}
+          type="newsletter"
```

Comment:
Since we're passing the type here, we should update the `PostLayout` component to not show an image for newsletter posts (since they don't have one).

Can you update `sites/org/components/layouts/post.mjs` so that line 93 looks like this:

```mjs
{type === 'newsletter' 
  ? null 
  : <PostImage imgId={`${type}-${dir}`} frontmatter={frontmatter} />
}
```

That will keep the (currently broken) image out of the view.

Apart from that, this is great.

Reply:
Done!